### ISSUE TITLE
NullPointerException when generating overview summary

### ISSUE THREAD

**Issue by lennartj** [grahamedgecombe/apiviz#1]
When this occurs, no JavaDoc nor Test JavaDoc is emitted.

```
[ERROR] Error while creating javadoc report:
Exit code: 1 - javadoc: error - An error occurred during diagram generation: java.lang.NullPointerException
java.lang.NullPointerException
    at org.jboss.apiviz.ClassDocGraph.isHidden(ClassDocGraph.java:314)
    at org.jboss.apiviz.ClassDocGraph.addPackageDependencies(ClassDocGraph.java:305)
    at org.jboss.apiviz.ClassDocGraph.getOverviewSummaryDiagram(ClassDocGraph.java:182)
    at org.jboss.apiviz.APIviz.generateOverviewSummary(APIviz.java:188)
    at org.jboss.apiviz.APIviz.start(APIviz.java:68)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.sun.tools.javadoc.DocletInvoker.invoke(DocletInvoker.java:310)
    at com.sun.tools.javadoc.DocletInvoker.start(DocletInvoker.java:189)
    at com.sun.tools.javadoc.Start.parseAndExecute(Start.java:366)
    at com.sun.tools.javadoc.Start.begin(Start.java:219)
    at com.sun.tools.javadoc.Start.begin(Start.java:205)
    at com.sun.tools.javadoc.Main.execute(Main.java:64)
    at com.sun.tools.javadoc.Main.main(Main.java:54)

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_92.jdk/Contents/Home/jre/../bin/javadoc @options @packages
```

**Comment by grahamedgecombe** [comment]
Are you able to share the source code that's causing this problem to help me reproduce it?

**Comment by trajano** [comment]
From the code I think it's this line 

https://github.com/grahamedgecombe/apiviz/blob/master/src/main/java/org/jboss/apiviz/ClassDocGraph.java#L314

that may require a null check.  I am thinking

     if (node.tags(TAG_HIDDEN).length > 0) {

should be

     if (node.tags(TAG_HIDDEN) != null && node.tags(TAG_HIDDEN).length > 0) {

**Comment by grahamedgecombe** [comment]
I've cherry-picked trajano's fix and included it in 1.3.4.
